src/Pages/Login/Login.page.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useContext } from 'react';
+import Login from '../../components/Login/Login.component';
+import { StoreContext } from '../../store/StoreContext';
+
+function LoginPage() {
+  const { store } = useContext(StoreContext);
+  const { authState } = store;
+
+  return (
+    <>
+      <h1 className="centered">Login</h1>
+      {!authState && <Login />}
```

Comment:
I think the authState and showing or not this page or the Register should be handled in the routes. As you should not enter the login or register if already logged in